czech/kapsy/nuz/riz.md

```diff
@@ -3,3 +3,5 @@ Z rozespalosti to trochu přeženeš.
 Zabereš víc, než jsi měl v úmyslu a málem si uřežeš poslední článek ukazováčku.
 
 K nechutně sladkému smradu marshmelounů se připojí nasládlý pach čerstvé krve.
+
+["Honem chňapneš jednoho maršmeláka, uděláš v něm díru, nasadíš ho na krvácející prst a zapalovačem stmelíš"](bandaz/marsmelak.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.